stories/molecules/components/networkGraph/tooltipExtractors.ts

```diff
@@ -0,0 +1,64 @@
+import type { Node } from 'ngraph.graph'
+import type { DeepReadonly } from 'superTypes'
+import type { TooltipExtractor } from 'ui/molecules/networkGraph/ecs/system/nodeTooltipSystem'
+import type { EvmosValidator, FamilyTreeObject } from './meshFactory'
+import femaleIcon from '../../../assets/female.svg'
+import maleIcon from '../../../assets/male.svg'
+
+export const familyTreeTooltipExtractor: TooltipExtractor<FamilyTreeObject> = (
+  node: DeepReadonly<Node<FamilyTreeObject>>
+) => {
+  const tooltip = document.createElement('div')
+  const ImgHTML = document.createElement('img')
+  const title = document.createElement('h3')
+  const datesWrapper = document.createElement('div')
+  const dateBirth = document.createElement('p')
+  const dateDeath = document.createElement('p')
+  const { name, gender, dateOfBirth, dateOfDeath, portrait }: FamilyTreeObject = node.data
+
+  const formatDate = (dateString: string): string => {
+    const [day, month, year]: string[] = dateString.split('/')
+    const date = new Date(`${year}-${month}-${day}`)
+    return date.toLocaleString('en-GB', {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric'
+    })
+  }
+
+  tooltip.classList.add('networkgraph-story-tooltip')
+  datesWrapper.classList.add('networkgraph-story-dates-wrapper')
+  ImgHTML.classList.add('networkgraph-story-portrait')
+
+  ImgHTML.src = portrait !== '' ? portrait : gender === 'male' ? maleIcon : femaleIcon
```

Comment:
I find it would be better to only have the people pictures concerning the binary tree (and to remove male/female svgs) => from a purely aesthetic point of view .

Reply:
I wouldn't say it makes much difference if there are the svgs or not. At least they reinforce the understanding of the graph (Spheres are female nodes).

I prefer to keep as is and maybe in the future we could find a more pertinent use case for the binary tree.